src/lib/modules/scaffolding/options.ts

```diff
@@ -0,0 +1,33 @@
+import Ajv from "ajv";
+
+import { Logger } from "../../../typings/logger";
+
+export enum Framework {
+  React = "react",
+}
+
+export enum ContractLanguage {
+  Solidity = "solidity",
+}
+
+export class Options {
```

Comment:
`Options` is also very very ambiguous here. Does anything speak against make those a bit more concrete?